package_test.go

```diff
@@ -819,7 +819,23 @@ func (s *PackageSuite) TestRun(c *C) {
 	var jimCheck = Person{}
 	err = db.Query(nil, selectStmt, &jim).Get(&jimCheck)
 	c.Assert(err, IsNil)
-	c.Assert(jimCheck, Equals, jim)
+
```

Comment:
It would be good to dogfood the new syntax in the `personAndAddressDB` function as well.

Reply:
Good idea! Done in the last commit!